### ISSUE TITLE
Replace `conditionals` with `conditionals-if`

### ISSUE THREAD

**Issue by jmrunkle** [exercism/java#1994]
This issue describes how to implement the `conditionals` concept exercise for the Java track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of Conditionals in Java. 

TODO: more description of this concept

## Learning objectives

- TODO

## Out of scope

- TODO

## Concepts

This Concepts Exercise's Concepts are:

- TODO

## Prequisites

This Concept Exercise's prerequisites Concepts are:

- TODO

### Hints

Nothing to report

### After

Nothing to report

## Representer

This exercise does not require any specific representation logic to be added to the [representer](https://github.com/exercism/java-representer).

## Analyzer

This exercise does not require any specific analyzer logic to be added to the [analyzer](https://github.com/exercism/java-analyzer).

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/java/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by jmrunkle** [comment]
FYI: this concept is blocking three other concept exercises (see https://github.com/exercism/java/issues/1992)

**Comment by jmrunkle** [comment]
Actually, it feels like conditionals is completely redundant with "conditionals-if" so I think we can probably just get rid of "conditionals" as a concept and use "conditionals-if" instead. @mirkoperillo - any thoughts or disagreement? If you agree, we can replace this issue with a PR to just remove / replace all of the references to the "conditional" with "conditional-if".

**Comment by mirkoperillo** [comment]
@jmrunkle I' agree with you
